File: @xoutput/api/src/websocket/emulation/ControllerInputResponse.ts
import { MessageBase } from '../MessageBase';
import { ControllerSourceValue } from './ControllerSourceValue';
import { ControllerTargetValue } from './ControllerTargetValue';

export const ControllerInputResponseType = 'ControllerInputFeedback';

export interface ControllerInputResponse extends MessageBase<typeof ControllerInputResponseType> {
  sources: ControllerSourceValue[];
  targets: ControllerTargetValue[];
}

File: XOutput.Api/Websocket/Emulation/ControllerInputResponse.cs
using System.Collections.Generic;

namespace XOutput.Websocket.Emulation
{
    public class ControllerInputResponse : MessageBase
    {
        public const string MessageType = "ControllerInputFeedback";
        public List<ControllerSourceValue> Sources { get; set; }
        public List<ControllerTargetValue> Targets { get; set; }
        public ControllerInputResponse()
        {
            Type = MessageType;
        }
    }
}
